src/components/devs/addDeveloper.js

```diff
@@ -0,0 +1,150 @@
+    formData.append("name", name);
+    formData.append("image", image);
+    formData.append("description", description);
+    formData.append("title", title);
+    formData.append("salary_exp", salary);
+    formData.append("rating", rating);
+    formData.append("user_id", 1);
+
+    dispatch(addDev(formData));
+    Navigate("/developers");
+    window.location.reload();
+  };
+
+  // handle image upload
+  const handleUploadImage = (e) => {
+    setImage(e.target.value);
+  };
+
+  return (
+    <div className="add-dev-container">
+      <div className="login-box" onSubmit={submitHandle}>
+        <form>
+          <div className="user-box">
+            <input
+              className="form-control"
+              type="text"
+              required
+              id="name"
+              value={name}
+              onChange={(e) => {
+                setName(e.target.value);
+              }}
+            />
+            <label htmlFor="name">name</label>
+          </div>
+
+          <div className="user-box">
+            <input
+              type="text"
+              required
+              className="form-control"
+              id="photo"
+              onChange={handleUploadImage}
+              multiple
+            />
+            <label htmlFor="photo">Image</label>
+          </div>
+
+          <div className="user-box">
+            <input
+              type="text"
+              className="form-control"
+              required
+              id="title"
+              onChange={(e) => {
+                setTitle(e.target.value);
+              }}
+            />
+            <label htmlFor="title">title </label>
+          </div>
+
+          <div className="user-box">
+            <input
+              type="text"
+              required
+              className="form-control"
+              id="description"
+              onChange={(e) => {
+                setDescription(e.target.value);
+              }}
+            />
+            <label htmlFor="description">Description</label>
+          </div>
+
+          <div className="user-box">
+            <input
+              type="number"
+              required
+              id="salary"
+              className="form-control"
+              onChange={(e) => {
+                setSalary(e.target.value);
+              }}
+            />
+            <label htmlFor="salary">salary</label>
+          </div>
+
+          <div className="user-box">
+            <input
+              type="number"
+              required
+              id="rating"
+              minLength={1}
+              maxLength={5}
+              className="form-control"
+              onChange={(e) => {
+                setRating(e.target.value);
+              }}
+            />
+            <label htmlFor="rating">rating</label>
+          </div>
+          {/* eslint_disable_anchor-is-valid */}
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+          <a href="#">
```

Comment:
[OPTIONAL] Is there a cleaner way to write this?

Reply:
Hi @mcrd25 

Can you some us a cleaner way of doing this by recommending any video or documentation for us